src/Core/ServiceWrapper/CLI/InstallOption.cs

```diff
@@ -0,0 +1,135 @@
+﻿using CommandLine;
+using System;
+using System.Diagnostics;
+using System.Security.AccessControl;
+using System.Text;
+using winsw.Native;
+using WMI;
+
+namespace winsw.CLI
+{
+    [Verb("install", HelpText = "install the service to Windows Service Controller")]
+    public class InstallOption : CliOption
+    {
+        [Option('p', "profile", Required = false, HelpText = "Service Account Profile")]
```

Comment:
:warning: Breaking change: `CommandLineParser` is not going to support forward slashes.